<ui version="4.0" >
 <class>IdentitiesDlg</class>
 <widget class="QDialog" name="IdentitiesDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>442</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>5</hsizetype>
    <vsizetype>5</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Edit Identity</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../images/icons.qrc" >:/default/identity.png</iconset>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>8</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QComboBox" name="identityList" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>3</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="insertPolicy" >
        <enum>QComboBox::NoInsert</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editIdentitiesButton" >
       <property name="text" >
        <string>Edit...</string>
       </property>
       <property name="icon" >
        <iconset resource="../images/icons.qrc" >:/default/edit_user.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="generalTab" >
      <attribute name="title" >
       <string>&amp;General</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>8</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item row="0" column="0" >
          <widget class="QLabel" name="label" >
           <property name="text" >
            <string>Real Name</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" >
          <widget class="QLineEdit" name="identEdit" />
         </item>
         <item rowspan="2" row="0" column="1" >
          <widget class="QLineEdit" name="realNameEdit" />
         </item>
         <item rowspan="2" row="1" column="0" >
          <widget class="QLabel" name="label_2" >
           <property name="text" >
            <string>Ident</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="nickGroupBox" >
         <property name="title" >
          <string>Nicknames</string>
         </property>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>8</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QListWidget" name="nickList" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>7</hsizetype>
              <vsizetype>7</vsizetype>
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QPushButton" name="addNickButton" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>5</hsizetype>
                <vsizetype>0</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string>&amp;Add...</string>
              </property>
              <property name="icon" >
               <iconset resource="../images/icons.qrc" >:/default/edit_add.png</iconset>
              </property>
              <property name="iconSize" >
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="editNickButton" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>5</hsizetype>
                <vsizetype>0</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string>&amp;Edit...</string>
              </property>
              <property name="icon" >
               <iconset resource="../images/icons.qrc" >:/default/edit.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="delNickButton" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>5</hsizetype>
                <vsizetype>0</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string>De&amp;lete</string>
              </property>
              <property name="icon" >
               <iconset resource="../images/icons.qrc" >:/default/edit_remove.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" >
              <property name="margin" >
               <number>0</number>
              </property>
              <property name="spacing" >
               <number>6</number>
              </property>
              <item>
               <spacer>
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" >
                 <size>
                  <width>0</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QToolButton" name="upNickButton" >
                <property name="text" >
                 <string>...</string>
                </property>
                <property name="icon" >
                 <iconset resource="../images/icons.qrc" >:/default/1uparrow.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="downNickButton" >
                <property name="text" >
                 <string>...</string>
                </property>
                <property name="icon" >
                 <iconset resource="../images/icons.qrc" >:/default/1downarrow.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <spacer>
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" >
                 <size>
                  <width>0</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>20</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
      <widget class="QWidget" native="1" name="widget" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>0</y>
         <width>395</width>
         <height>83</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="awayTab" >
      <attribute name="title" >
       <string>&amp;Away</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>8</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item row="0" column="1" >
          <widget class="QLineEdit" name="awayNickEdit" />
         </item>
         <item row="1" column="0" >
          <widget class="QCheckBox" name="enableAwayReason" >
           <property name="text" >
            <string>Away Reason</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QLineEdit" name="awayReasonEdit" />
         </item>
         <item row="2" column="0" >
          <widget class="QCheckBox" name="enableReturnMessage" >
           <property name="text" >
            <string>Return Message</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" >
          <widget class="QLineEdit" name="returnMessageEdit" />
         </item>
         <item row="0" column="0" >
          <widget class="QCheckBox" name="enableAwayNick" >
           <property name="text" >
            <string>Away Nick</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>Auto Away</string>
         </property>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>8</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QCheckBox" name="enableAutoAway" >
            <property name="text" >
             <string>Enable Auto Away</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="autoAwayLabel_1" >
              <property name="text" >
               <string>Auto away after</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="autoAwayTime" />
            </item>
            <item>
             <widget class="QLabel" name="autoAwayLabel_2" >
              <property name="text" >
               <string>minutes</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QGridLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item row="1" column="0" >
             <widget class="QCheckBox" name="enableAutoAwayReturn" >
              <property name="text" >
               <string>Return Message</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" >
             <widget class="QCheckBox" name="enableAutoAwayReason" >
              <property name="text" >
               <string>Away Reason</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1" >
             <widget class="QLineEdit" name="autoAwayReturnEdit" />
            </item>
            <item row="0" column="1" >
             <widget class="QLineEdit" name="autoAwayReasonEdit" />
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="messagesTab" >
      <attribute name="title" >
       <string>&amp;Messages</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>8</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item row="1" column="0" >
          <widget class="QLabel" name="label_3" >
           <property name="text" >
            <string>Part Reason</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QLineEdit" name="partReasonEdit" />
         </item>
         <item row="0" column="1" >
          <widget class="QLineEdit" name="kickReasonEdit" />
         </item>
         <item row="2" column="0" >
          <widget class="QLabel" name="label_5" >
           <property name="text" >
            <string>Quit Reason</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" >
          <widget class="QLineEdit" name="quitReasonEdit" />
         </item>
         <item row="0" column="0" >
          <widget class="QLabel" name="label_4" >
           <property name="text" >
            <string>Kick Reason</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>141</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../images/icons.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>enableAwayNick</sender>
   <signal>clicked(bool)</signal>
   <receiver>awayNickEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>71</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel" >
     <x>212</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enableAwayReason</sender>
   <signal>clicked(bool)</signal>
   <receiver>awayReasonEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>111</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel" >
     <x>153</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enableReturnMessage</sender>
   <signal>clicked(bool)</signal>
   <receiver>returnMessageEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>120</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel" >
     <x>166</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enableAutoAway</sender>
   <signal>clicked(bool)</signal>
   <receiver>autoAwayTime</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>79</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel" >
     <x>144</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enableAutoAwayReturn</sender>
   <signal>clicked(bool)</signal>
   <receiver>autoAwayReturnEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>121</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel" >
     <x>202</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enableAutoAwayReason</sender>
   <signal>clicked(bool)</signal>
   <receiver>autoAwayReasonEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>140</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel" >
     <x>169</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>IdentitiesDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>226</x>
     <y>420</y>
    </hint>
    <hint type="destinationlabel" >
     <x>226</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>IdentitiesDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>226</x>
     <y>420</y>
    </hint>
    <hint type="destinationlabel" >
     <x>226</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
